incubator/jenkins/.helmignore

```diff
@@ -0,0 +1,21 @@
+# Patterns to ignore when building packages.
```

Comment:
The .helmignore file gets add during the `helm create` process. See the workflow below:
```
$ bin/helm create something
Creating something
$ cd something/
$ ls -a
.      		.helmignore    	charts 		values.yaml
..     		Chart.yaml     	templates
$ cat .helmignore
# Patterns to ignore when building packages.
# This supports shell glob matching, relative path matching, and
# negation (prefixed with !). Only one pattern per line.
.DS_Store
# Common VCS dirs
.git/
.gitignore
.bzr/
.bzrignore
.hg/
.hgignore
.svn/
# Common backup files
*.swp
*.bak
*.tmp
*~
# Various IDEs
.project
.idea/
*.tmproj
```

Is there something further than you are suggesting? @chrislovecnm @viglesiasce